cli/interaction.ss

```diff
@@ -267,13 +268,12 @@
              ;; enum off-chain-channel = 'stdio | 'libp2p
              (option 'off-chain-channel-selection "-C" "--off-chain-channel" default: 'stdio
                      help: "command to specify off-chain-channel")
-             (option 'host-address "-O" "--host-address" default: #f
-                     help: "host-address (only required if using libp2p as off-chain-channel)")
-             ;; FIXME: This should be stored and extracted from `contacts`.
-             ;; It is supplied here as a temporary workaround,
-             ;; until storing peerIds in `contacts` is supported.
-             (option 'dest-address "-d" "--dest-address" default: #f
-                     help: "dest-address (only required if using libp2p as off-chain-channel)")
+             (option 'host-address "-O" "--host-address" default: "/ip4/0.0.0.0/tcp/10333"
+                     help: "host-address for libp2p (default value is /ip4/0.0.0.0/tcp/10333)")
```

Comment:
I'm pretty sure the cli library we're using will display the defaults on its own (do `glow help start-interaction` to check).

Reply:
Awesome! Just made some changes addressing this